src/app/random-joke/random-joke.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 class="btn-center">Random Joke</h1>
```

Comment:
This component view and the jokes component view has identical elements. Can it be refactored such that there is more reusability?

Reply:
how can i refactor those,
because in joke component i have used the for loop so that i will get all Joke list.
but in random joke i want only one list...